technique/reflectometry/Runscript_day2.py

```diff
@@ -0,0 +1,228 @@
+    run_angle(sample_3, 2.3, 20)
+    inject(sample_3, SiMW, flow=1, volume=18, wait=False)
+
+    # === SiMW Measurements ===
+    sample_1.subtitle = '10C SiMW'
+    run_angle(sample_1, 0.7, 15)
+    run_angle(sample_1, 2.3, 25)
+    inject(sample_1, H2O, flow=1, volume=18, wait=False)
+
+    sample_3.subtitle = '38C + MPB Flush SiMW'
+    run_angle(sample_3, 0.7, 15)
+    run_angle(sample_3, 2.3, 25)
+    inject(sample_3, H2O, flow=1, volume=18, wait=False)
+
+    # === H2O Measurements ===
+    sample_1.subtitle = '10C H2O'
+    run_angle(sample_1, 0.7, 10)
+    run_angle(sample_1, 2.3, 25)
+    inject(sample_1, D2O, flow=1, volume=18, wait=False)
+
+    sample_3.subtitle = '38C + MPB Flush H2O'
+    run_angle(sample_3, 0.7, 10)
+    run_angle(sample_3, 2.3, 25)
+    inject(sample_3, D2O, flow=1, volume=18, wait=False)
+
+    # === Protein injections ===
+
+    inject(sample_1, "SYRINGE_1", flow=1, volume=7, wait=True)
+    sample_1.subtitle = '10C + MPB incubation D2O'
+    run_angle(sample_1, 0.7, count_seconds=900)
+    inject(sample_1, D2O, flow=1, volume=18, wait=True)
+
+    # === D2O Measurements ===
+    sample_1.subtitle = '10C + MPB Flush D2O'
+    run_angle(sample_1, 0.7, 7)
+    run_angle(sample_1, 2.3, 20)
+    inject(sample_1, FourMW, flow=1, volume=18, wait=False)
+
+    sample_2.subtitle = '10C + MPB Flush D2O'
+    run_angle(sample_2, 0.7, 7)
+    run_angle(sample_2, 2.3, 20)
+    inject(sample_2, FourMW, flow=1, volume=18, wait=False)
+
+    # === 4MW Measurements ===
+    sample_1.subtitle = '10C + MPB Flush 4MW'
+    run_angle(sample_1, 0.7, 7)
+    run_angle(sample_1, 2.3, 20)
+    inject(sample_1, SiMW, flow=1, volume=18, wait=False)
+
+    sample_2.subtitle = '10C + MPB Flush 4MW'
+    run_angle(sample_2, 0.7, 7)
+    run_angle(sample_2, 2.3, 20)
+    inject(sample_2, SiMW, flow=1, volume=18, wait=False)
+
+    # === SiMW Measurements ===
+    sample_1.subtitle = '10C + MPB Flush SiMW'
+    run_angle(sample_1, 0.7, 15)
+    run_angle(sample_1, 2.3, 25)
+    inject(sample_1, H2O, flow=1, volume=18, wait=False)
+
+    sample_2.subtitle = '10C + MPB Flush SiMW'
+    run_angle(sample_2, 0.7, 15)
+    run_angle(sample_2, 2.3, 25)
+    inject(sample_2, H2O, flow=1, volume=18, wait=False)
+
+    # === H2O Measurements ===
+    sample_1.subtitle = '10C + MPB Flush H2O'
+    run_angle(sample_1, 0.7, 15)
+    run_angle(sample_1, 2.3, 25)
+    inject(sample_1, D2O, flow=1, volume=18, wait=False)
+
+    sample_2.subtitle = '10C + MPB Flush H2O'
+    run_angle(sample_2, 0.7, 15, vgaps={"S2VG": 2.0, "S3VG": 3.0})
+    run_angle(sample_2, 2.3, 25)
+    inject(sample_2, D2O, flow=1, volume=18, wait=False)
+
+    # KEEP THIS:
+    if dry_run:
+        run_summary()
+        # return DryRun.run_time
+
+
+runscript(dry_run=True)
+
+ans = input("Script will run for " +
+            str(int(DryRun.run_time / 60)) + "h " + str(int(DryRun.run_time % 60)) + "min ===\nContinue ([y]/n)?")
+
+if ans == 'y' or ans == "":
```

Comment:
may as well use ans.lower() here to catch if someone puts "Y" instead of "y"

Reply:
Agreed. This is however just a user test script, so not critical.